include/swift/Basic/Features.def

```diff
@@ -553,6 +553,9 @@ EXPERIMENTAL_FEATURE(BorrowAndMutateAccessors, false)
 /// Allow use of @inline(always) attribute
 SUPPRESSIBLE_EXPERIMENTAL_FEATURE(InlineAlways, false)
 
+/// Emit experimental perfomance hints for low-level (embedded) Swift code
+EXPERIMENTAL_FEATURE(SwiftPerformanceHints, true)
```

Comment:
You shouldn't need an experimental feature it. Performance hints are opt-in warnings. If you add the overall `PerformanceHints` warning group, then `-Wwarning PerformanceHints` or `-Werror PerformanceHints` will Just Work™️ to enable them as warnings or errors.

Reply:
Removed, thanks!